array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,12 @@
+/* exported equal */
+// create new function equal with 2 parameters, first and second
+// convert array to JSON string and check if they're equal
+
+function equal(first, second) {
+  if (JSON.stringify(first) === JSON.stringify(second)) {
+    return true;
+  }
+  if (JSON.stringify(first) !== JSON.stringify(second)) {
+    return false;
+  }
```

Comment:
Try doing this without `JSON.stringify()`. It's really inefficient for array comparison.